src/perf/impeller.md

```diff
@@ -1,152 +1,111 @@
-they won't be compiled while an app is running.
+at Engine build time so they don't compile at runtime.
 
 [early-onset jank]: {{site.github}}/flutter/flutter/projects/188
 [smaller, simpler set of shaders]: {{site.github}}/flutter/flutter/issues/77412
 
 For a video introduction to Impeller, check out the following
-talk from the Flutter Foward 2023 conference:
+talk from the Flutter Foward 2023 conference.
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/gKrYWC_SDxQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 [What is Impeller?](https://www.youtube.com/watch?v=gKrYWC_SDxQ)
 
-Overall, Impeller has the following objectives:
+Impeller has the following objectives:
 
 * **Predictable performance**:
-  All shader compilation and reflection is performed
-  offline at build time. All pipeline state objects
-  are built upfront. Caching is explicit and under
-  the control of the engine.
-* **Instrumentable**: All graphics resources
-  (textures, buffers, pipeline state objects, and so on)
-  are tagged and labeled. Animations can be captured
-  and persisted to disk without affecting per-frame
-  rendering performance.
-* **Portable**: Not tied to a specific client rendering API.
-  Shaders are authored once and converted to
-  backend-specific formats as necessary.
-* **Uses modern graphics APIs effectively**:
-  Makes heavy use of (but doesn’t depend on)
-  features available in Modern APIs like Metal and Vulkan.
-* **Makes effective use of concurrency**:
-  Can distribute single-frame workloads across
-  multiple threads if necessary.
+  Impeller compiles all shaders and reflection offline at build time.
+  It builds all pipeline state objects upfront.
+  The engine controls caching and caches explicitly.
+* **Instrumentable**:
+  Impeller tags and labels all graphics resources like textures,
+  and buffers.
+  It can capture and persist animations to disk without affecting
+  per-frame rendering performance.
+* **Portable**:
+  Flutter doesn't tie Impeller to a specific client rendering API.
+  You can author shaders once and convert them to backend-specific
+  formats as necessary.
+* **Leverages modern graphics APIs**:
+  Impeller uses, but doesn’t depend on, features available in
+  modern APIs like Metal and Vulkan.
+* **Leverages concurrency**:
+  Impeller can distribute single-frame workloads across multiple
+  threads if necessary.
 
 ## Availability
 
-Where can you currently use Impeller?
+Where can you use Impeller?
 
 ### iOS
 
-Impeller is currently available for preview on iOS.
-You can enable it by passing `--enable-impeller`
-to the `flutter run` command or add the
-following under the top-level `<dict>` tag in an
-application's `Info.plist` file:
+Flutter enables Impeller by default on iOS.
+
+To _disable_ Impeller on iOS, take one of the following actions:
```

Comment:
Worth noting that --disable-impeller is only for debugging and if you want to _ship_ with impeller off you must add it to the Plist entry.